src/main/resources/marc4bibframeAutority.yml

```diff
@@ -0,0 +1,65 @@
+authorityFieldRules:
+  '100':
+    - types: CONCEPT, PERSON
+      include: 100_mappings
+      marc2ldCondition:
+        ind1: '!3'
+      edges:
+        - types: PERSON
+          predicate: FOCUS
+          subfields:
+            a: NAME
+            b: NUMERATION
+            c: TITLES
+            d: DATE
+            j: ATTRIBUTION
+            q: NAME_ALTERNATIVE
+    - types: CONCEPT, FAMILY
+      include: 100_mappings
+      marc2ldCondition:
+        ind1: '3'
+      edges:
+        - types: FAMILY
+          predicate: FOCUS
+          subfields:
+            a: NAME
+            b: NUMERATION
+            c: TITLES
+            d: DATE
+            j: ATTRIBUTION
+            q: NAME_ALTERNATIVE
+
+
+sharedAuthorityRules:
+  100_mappings:
+    predicate: CLASSIFICATION
```

Comment:
`predicate: CLASSIFICATION` is not needed here.
This is the root resource. There is no predicate connecting to this resouce.